src/components/select/select.scss

```diff
@@ -52,10 +52,6 @@
   );
 }
 
-:host([disable-animation]) {
-  --sbb-options-panel-animation-duration: 0.1ms;
```

Comment:
Is the new logic missing here?

Reply:
It is handled in `overlay.scss` now since it's shared between the autocomplete and the select